simulate_robot_index: 1
direction_weight: 1
distance_weight: 6.89992e-310
other_robot_weight: 1
